payload/src/dynamics.ts

```diff
@@ -78,6 +77,19 @@ export function _OnLoad() {
         if (action && action != "") {
             Action(element as HTMLElement, action)
         }
+	});
+
+	document.querySelectorAll('[ssa\\\\:poll]').forEach(element => {
```

Comment:
It would be nice to have separate function that will enable polling for components and keep `_OnLoad` just for `ssa:onload`.

Reply:
good thinking, done.